compositor_render/src/transformations/shader/validation/shader_header_web.wgsl

```diff
@@ -0,0 +1,16 @@
+struct VertexInput {
+    @location(0) position: vec3<f32>,
+    @location(1) tex_coords: vec2<f32>,
+}
+
+struct BaseShaderParameters {
+    plane_id: i32,
+    time: f32,
+    output_resolution: vec2<u32>,
+    texture_count: u32,
```

Comment:
I'm not sure if `texture_count` is needed here. Currently, we only allow one texture but in the future we might support more